### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by aristizabal95** [OpenMined/courses#16] [Introduce Yourself]
Greetings from Colombia! 🇨🇴

I'm Alejandro Aristizabal and I'm a self-taught developer with a huge passion about AI and Machine Learning! I've been learning and practicing the field for about 3-4 years now and I'm always amazed at the technology that arises thanks to AI.

I participated in the Secure & Private AI Challenge, and that's how I learned about Private AI and the OpenMIned team.

I'm happy I get to participate on this course and learn a lot from you guys.  😁

**Comment by yemikifouly** [comment]
Alejandro!!! 🙂 Hi! So good that you have enrolled in the course! It is such a huge step up from the SPAI Couse. I am sure you will enjoy it a lot!

**Reply by aristizabal95** [reply]
I sure will! Had a great time with the SPAI Course, I can't wait to see what the team has prepared for this course! I expect my mind to be blown more than once 🤯.

Thank you and everyone else in the OpenMined team for putting this together and releasing it for free to the public. It's truly amazing!
